cli/types/cypress.d.ts

```diff
@@ -228,6 +228,11 @@ declare namespace Cypress {
      */
     LocalStorage: LocalStorage
 
+    /**
+     * Fire automation:request event for internal use.
+     */
+    automation(eventName: string, ...args: any[]): Promise<void>
```

Comment:
I believe `automation` is able to return data, so let's open this a little bit:

```suggestion
    automation(eventName: string, ...args: any[]): Promise<any>
```